assets/queries/k8s/app_armor_config/test/positive_expected_result.json

```diff
@@ -0,0 +1,22 @@
+[
+	{
+		"queryName": "app_armor_config",
```

Comment:
The query name should explain the vulnerability. Try something like "Missing_App_Armor_Config"